src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,77 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int buySum = 0;
+    private int supplySum = 0;
+    private final String supplyOperation = "supply";
+    private final String buyOperation = "buy";
+    private final String result = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] tempString = readFile(fromFileName).toString().split("\\W+");
```

Comment:
1. use more meaningful name for variable
2. create string array like  ["supply,100", "buy,100"...]. with such an approach code will work even if position of array elements was changed

Reply:
To be honest, I don't understand what you mean? Could you please explain? Because I received the terms of reference, where the data presentation template was given. And I followed the pattern. If you think that the order of values can change, then there can be a lot of options, for example:
- supply,100
- 100,suplly
- supply,buy,100,200
-...

Reply:
- "create string array like ["supply,100", "buy,100"...]" - I can do it, but then in the calculateSum method I have to do the same - to split the array, like I already did here:
String[] tempString = readFile(fromFileName).toString().split(System.lineSeparator());